<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModFilterWidget</class>
 <widget class="QTabWidget" name="ModFilterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <widget class="QWidget" name="VersionPage">
   <attribute name="title">
    <string>Minecraft versions</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QFormLayout" name="formLayout">
      <item row="2" column="0">
       <widget class="QRadioButton" name="allVersionsButton">
        <property name="text">
         <string notr="true">allVersions</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="strictVersionButton">
        <property name="text">
         <string notr="true">strictVersion</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="majorVersionButton">
        <property name="text">
         <string notr="true">majorVersion</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
